middleman/chat_functions.py

```diff
@@ -88,7 +90,16 @@ async def send_text_to_room(
         logger.exception(f"Unable to send message response to {room_id}")
         return f"Failed to send message: {ex}"
 
-
+async def get_dm(client: AsyncClient, user: str): 
+    room_id = self.store.get_dm(user_id=user)
+    if room_id == "0":
+        resp = client.room_create(
```

Comment:
```suggestion
        resp = await client.room_create(
```

Pretty sure this should be awaited?